### ISSUE TITLE
Confused with rn-camera-roll plugin? Can anyone help?

### ISSUE THREAD

**Issue by Msspl-PrashenjeetRoy** [bamlab/rn-camera-roll#13]
Can you please tell me all the features of your plugin and what are future implementations. It will really helpful.
I am able to **access gallery photos>images.** But what next how can I select those images by clicking. Is there any options in your plugin or I need to implement it myself?

For now I want to know these points:
1. Can I select multiple images for upload? if yes how or where I need to go?
2. Can I select video?

Basically I want to make a fb like posting with multiple image upload. So after opening gallery I need to **select multiple/single images > show all uploaded items > upload button which actually uploads it to feed/wall (as fb did after multiple image upload)** . @bruz

**Comment by 4ian** [comment]
Hey! This module was a kind of polyfill for when CameraRoll API from React Native was only supported on iOS. It's now stable and fully available on both iOS and Android.
I've updated the message in the README to make it bigger and clearer that this module is deprecated: https://github.com/bamlab/rn-camera-roll#as-of-react-native-019-the-cameraroll-api-is-now-available-on-android-too-you-should-not-use-this-module-anymore-and-only-use-the-cameraroll-api-from-react-native

This plugin does nothing more than the CameraRoll API and won't be developed anymore. 
It's not an image picker as you can see by looking the usage section of the README: https://github.com/bamlab/rn-camera-roll#usage
Please note that the CameraRoll API from React Native is not a fully featured image picker too, it's just offering a way to fetch the images from the Camera gallery of the phone.

If you want a ready made image picker for React Native, take a look at this one: https://github.com/marcshilling/react-native-image-picker or https://github.com/ivpusic/react-native-image-crop-picker or maybe something else :) (I've just made a quick search for "react native image picker"). https://js.coach/react-native is also a good resource for ready made module.

**Comment by Msspl-PrashenjeetRoy** [comment]
I already have those links of other image picker plugin now giving a try to this one https://github.com/ivpusic/react-native-image-crop-picker. Thanks for clearing things.
